src/safeLogger.ts

```diff
@@ -1,21 +1,22 @@
 import {
-  Logger,
-  LogLevel
+  Logger
 } from './types';
 
-type safeLogger = (loglevel: LogLevel) => (message: string, logger: Logger | undefined) => void;
-const safeLogger: safeLogger = (loglevel) => (message, logger) =>
-  logger && logger[loglevel] && logger[loglevel].call(logger, message);
+const safeCall = (prop: string, obj?: any) =>
```

Comment:
1) Does it make sense to call function without second parameter `('foo')`
Since any includes undefined, I think the parameter shouldn't be optional.
2) `safeCall` isn't the best naming imho, since there is nothing called.

Reply:
1. the user is allowed to pass `undefined` as a logger which is then passed here so here it has also to be allowed.
2. I agree. ^^

Reply:
1. `obj: any` statisfies already undefined.